projects/usingSoftTimers/src/main.c

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
#pragma once is only used in header files; this is to prevent headers from being included multiple times. It's unnecessary for .c files.